index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>John Thiong'o Murianki</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap' rel='stylesheet'>
+</head>
+<body>
+    <div id ="content">
+    <div id ="status-bar"></div>
+    <header>
+        <a id="logo">
+            <img src="/media/My Logo.png">
+        </a>
+        <nav>
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+        </nav>
+    </header>
+    <section id="greeting">
+        <p>I'm John Glad to see you</p>
+    </section>
+    <section id="about">
+        <p>
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+    </section>
+    <section id="socials">
+        <h2 id="connect">LET'S CONNECT</h2>
+        <div id="social-icons"> 
+            <img src="/media/social media icons.png">
+        </div>
+    </section>
+    <!--<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Projects</title>
+</head>
+<body>
+<div id ="project">
+    <img src="media/Snapshoot Portfolio.png">
+</div>
+<div id ="project-title">
+    Tonic
+</div>
+<div id="project-description"></div>
+<div id="project-detail"></div>
+<div id="languages-used"></div>
+<div id="view-project"></div>
+
+</body>
+</html>
+-->
```

Comment:
- [ ] As a general requirement you shouldn't commit commented-out code. It distracts the programmer who is reading your code and tempts them to try to figure out why the code was commented on and what it would do if it weren't. Additionally, it doesn't make the code looks neat :+1:. Here you can find more information about that ["The danger of commenting out code"](https://www.markhneedham.com/blog/2009/01/17/the-danger-of-commenting-out-code/)